src/test/groovy/graphql/nadel/engine/NadelExecutionStrategyTest3.groovy

```diff
@@ -0,0 +1,338 @@
+package graphql.nadel.engine
+
+import graphql.ExecutionInput
+import graphql.GraphQLError
+import graphql.execution.ExecutionId
+import graphql.execution.nextgen.ExecutionHelper
+import graphql.nadel.*
+import graphql.nadel.dsl.ServiceDefinition
```

Comment:
.* imports are bad

Reply:
darn auto-optimize imports. thought i disabled it